Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -1,37 +1,301 @@
+            {
+                return InValidOutput;
+            }
+
+            var HighestBalance = decimal.MinValue;
+            var RichestPerson = new StringBuilder();
+
+            for (int i = 0; i <= peopleAndBalances.Length - 1; i++)
+            {
+                var currentPersonData = peopleAndBalances[i].Split(',');
+
+                if (!decimal.TryParse(currentPersonData[currentPersonData.Length - 1], out decimal highestAmountOfPerson))
+                {
+                    return "N/A.";
+                }
+
+                if (highestAmountOfPerson == HighestBalance)
+                {
+                    if (i == peopleAndBalances.Length - 1)
+                    {
+                        RichestPerson.Append(" and ");
+                    }
+                    else
+                    {
+                        RichestPerson.Append(", ");
+                    }
+
+                    RichestPerson.Append(currentPersonData[0]);
+                }
+
+                HighestBalance = FindMorePeopleWithSameAmount(HighestBalance, RichestPerson, currentPersonData, highestAmountOfPerson);
+
+            }
+
+            var answer = $"{RichestPerson.ToString()} is the richest person. ¤{HighestBalance}.";
+
+            return answer;
+
+
+        }
+
+        private static decimal FindMorePeopleWithSameAmount(decimal HighestBalance, StringBuilder RichestPerson, string[] currentPersonData, decimal highestAmountOfPerson)
+        {
+            if (highestAmountOfPerson > HighestBalance)
+            {
+                HighestBalance = highestAmountOfPerson;
+                var currentOne = RichestPerson.ToString();
+
+                if (string.IsNullOrEmpty(currentOne))
+                {
+                    RichestPerson.Append(currentPersonData[0]);
+                }
+                else
+                {
+                    RichestPerson.Replace(currentOne, currentPersonData[0]);
+                }
+
+
+            }
+
+            return HighestBalance;
         }
 
         /// <summary>
         /// Return name and current money of the most poor person.
         /// </summary>
         public static string FindMostPoorPerson(string[] peopleAndBalances)
         {
-            return "";
+            if (IsInValidValidInput(peopleAndBalances))
+            {
+                return InValidOutput;
+            }
+
+            var lowestBalance = int.MaxValue;
+            var poorestPerson = new StringBuilder();
+
+
+            for (int i = 0; i <= peopleAndBalances.Length - 1; i++)
+            {
+                var currentPersonData = peopleAndBalances[i].Split(',');
+                if (!int.TryParse(currentPersonData[currentPersonData.Length - 1], out int highestAmountOfPerson))
+                {
+                    return "N/A.";
+                }
+
+                if (highestAmountOfPerson == lowestBalance)
+                {
+                    if (i == peopleAndBalances.Length - 1)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the '?:' operator here.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4937470)